qcengine/programs/nwchem/harvester.py

```diff
@@ -89,7 +89,15 @@ def harvest_outfile_pass(outtext):
             # logger.debug (mobj.group(1))
             psivar['NUCLEAR REPULSION ENERGY'] = mobj.group(1)
 
+        # Process DFT dispersion energy (a.u.)
+        mobj = re.search(r'^\s+' + r'(?:Dispersion correction)' + r'\s+=\s*' + NUMBER + r'\s*$', outtext, re.MULTILINE)
+        if mobj:
+            logger.debug('matched Dispersion')
+            logger.debug(mobj.group(1))
+            psivar['DFT DISPERSION ENERGY'] = mobj.group(1)
```

Comment:
Looks great! Can you add this to the `AtomicResult.properties` field as well from [here](http://docs.qcarchive.molssi.org/projects/QCElemental/en/stable/model_result.html#qcelemental.models.results.AtomicResultProperties).  The keyword is `scf_dispersion_correction_energy`.